tests/@angular-devkit/build-optimizer/webpack/aio-app/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "webpack-app",
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "build": "rimraf dist && webpack --hide-modules",
+    "build-no-ngo": "rimraf dist-no-ngo && webpack --config=webpack.config.no-ngo.js --hide-modules",
+    "build-old-ngo": "rimraf dist-old-ngo && webpack --config=webpack.config.old-ngo.js --hide-modules",
+    "serve": "lite-server",
+    "pree2e": "npm run build -- --devtool=source-map",
+    "e2e": "concurrently \"npm run serve\" \"npm run protractor\" --kill-others --success first",
+    "preprotractor": "webdriver-manager update",
+    "protractor": "protractor protractor.config.js",
+    "benchmark": "npm run build && npm run build-no-ngo && npm run build-old-ngo && node benchmark.js && node parity-benchmark.js",
+    "reinstall-bo": "rimraf node_modules/build-optimizer && npm i ../../../../../dist/@angular-devkit/build-optimizer/"
+  },
+  "dependencies": {
+    "@angular/animations": "4.2.3",
+    "@angular/common": "4.2.3",
+    "@angular/compiler": "4.2.3",
+    "@angular/compiler-cli": "4.2.3",
+    "@angular/core": "4.2.3",
+    "@angular/forms": "4.2.3",
+    "@angular/http": "4.2.3",
+    "@angular/material": "2.0.0-beta.7",
+    "@angular/platform-browser": "4.2.3",
+    "@angular/platform-browser-dynamic": "4.2.3",
+    "@angular/router": "4.2.3",
+    "@angular/service-worker": "1.0.0-beta.16",
+    "@ngtools/webpack": "1.4.1",
+    "core-js": "^2.4.1",
+    "rxjs": "^5.1.0",
+    "zone.js": "^0.8.4"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-optimizer": "file:../../../../../dist/@angular-devkit/build-optimizer",
+    "@types/jasmine": "2.5.45",
+    "compression-webpack-plugin": "^0.4.0",
+    "concurrently": "^3.4.0",
+    "copy-webpack-plugin": "^4.0.1",
+    "jasmine": "^2.6.0",
+    "lite-server": "^2.3.0",
+    "ngo-loader": "alxhub/ngo",
+    "protractor": "^5.1.2",
+    "purify": "igorminar/purify",
```

Comment:
Purify and ngo-loader are needed for the parity test, otherwise I can't test if build-optimizer is performing at least as well.